app/components/Payment/Quote.tsx

```diff
@@ -1,100 +1,181 @@
-import { type BuildSwapTransaction, fetchBuildExecuteTransaction, fetchQuotes, type Quote, type QuoteRequest } from "@avnu/avnu-sdk";
-import { useAccount, useContractWrite } from "@starknet-react/core";
 import { useEffect, useMemo, useState } from "react";
-import { num } from "starknet";
-import { useNotifications } from "~/root";
-import { ETH_DECIMALS, USDC_DECIMALS } from "~/utils/constant";
 import { GreenButton } from "../Buttons/ActionButton";
+import { NotificationSource } from "~/utils/notifications/sources";
+import { TransactionStatus, num } from "starknet";
+import { useAccount, useContractWrite } from "@starknet-react/core";
+import { useNotifications } from "~/root";
+import { getStarkscanUrl } from "~/utils/utils";
+import { type BuildSwapTransaction, type Quote, type QuoteRequest, fetchBuildExecuteTransaction, fetchQuotes } from "@avnu/avnu-sdk";
+import _ from "lodash";
+
+const USDC_CONTRACT = "0x053c91253bc9682c04929ca02ed00b3e423f6710d2ee7e0d5ebb06f3ecf368a8";
```

Comment:
mainnet address

Reply:
It's a fake component. We can leave it here.